hooks/command

```diff
@@ -87,6 +95,71 @@ function tail_logs {
   echo "0" > "$log_completion_marker_file"
 }
 
+# External Secrets Handler
+if [[ -n $external_secrets || -n $external_secrets_list || -n $secret_store || -n $cluster_store ]]; then
+  echo "--- :kubernetes: Creating Secrets"
+  
+  # A ClusterStore and SecretStore cannot be both used in the same step
+  if [[ -n $secret_store && -n $cluster_store ]]; then
+    echo "A ClusterStore and SecretStore cannot be used in the same step.. Exiting."
+    exit 42
+  fi 
+  
+  # If a store is defined, make sure secrets are defined
+  if ([ -n $cluster_store ] || [ -n $external_secrets ]) && [ -z $external_secrets_list ]; then 
+    echo "A Store is defined, but no External Secrets are defined.. Exiting"
+    exit 42
+  fi
+
+  # If a secret is defined, make sure a secret store is also defined
+  if [[ -n $external_secrets && -z $secret_store && -z $cluster_store ]]; then
+    echo "External Secrets are defined, but no Secret Store is defined.. Exiting."
+    exit 42
+  fi 
```

Comment:
Nice! I wish we had a nice dry-run mode here because this seems like a good case for testing - there's f.ex. two different syntaxes for if-statements here (bash `[[` vs basic `[`). I'm not questioning whether it's correct - but it's a maintenance burden.